test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -55,36 +55,40 @@ string ProtoConverter::createAlphaNum(string const& _strBytes) const
 
 bool ProtoConverter::isCaseLiteralUnique(Literal const& _x)
 {
-	std::string tmp;
+	std::string caseLiteralValue;
 	bool isUnique = false;
 	bool isEmptyString = false;
+
 	switch (_x.literal_oneof_case())
 	{
 	case Literal::kIntval:
-		tmp = std::to_string(_x.intval());
+		caseLiteralValue = std::to_string(_x.intval());
```

Comment:
This logic seems a little complicated to me. Why don't you go straight away to u256 inside the switch cases? Then you would not need the additional `if` statement at the end.

Reply:
Simplified logic now.